src/components/LandingPage/data/platformData.tsx

```diff
@@ -113,4 +113,18 @@ export const PlatformList: CardItem[] = [
     time: "15 min",
     link: "/tutorials/platform/secrets-management",
   },
+  {
+    title: "Configure OIDC with GCP WIF for CI hosted builds",
```

Comment:
```suggestion
    title: "Configure OIDC with GCP WIF for Harness CI Cloud builds",
```

Please use "Harness Cloud" or "Harness CI Cloud" instead of hosted. Hosted is an internal term.

Reply:
updated